modules/programs/vim.nix

```diff
@@ -135,16 +160,16 @@ in
 
       vim = pkgs.vim_configurable.customize {
         name = "vim";
-        vimrcConfig.customRC = customRC;
-        vimrcConfig.vam.knownPlugins = pkgs.vimPlugins;
-        vimrcConfig.vam.pluginDictionaries = [
-          { names = defaultPlugins ++ cfg.plugins; }
-        ];
-      };
+        vimrcConfig = {
+          inherit customRC;
 
-    in mkIf cfg.enable {
-      programs.vim.package = vim;
-      home.packages = [ cfg.package ];
-    }
+          packages.home-manager.start = plugins;
```

Comment:
Where is `packages.home-manager.start` is defined?
Are we moving away from VAM?

Reply:
I followed the manual: https://nixos.org/nixpkgs/manual/#managing-plugins-with-vim-packages

Yes, the plugins are then managed via the built in package manager of vim.

Reply:
Nice!

Reply:
Is this change backwards compatible?

Reply:
I can't think of a way on how this could introduce problems as here is the only place, you can configure and setup packages